.github/workflows/sycl-linux-precommit.yml

```diff
@@ -89,6 +89,21 @@ jobs:
             reset_intel_gpu: true
             install_drivers: ${{ contains(needs.detect_changes.outputs.filters, 'drivers') }}
             extra_lit_opts: --param gpu-intel-gen12=True
+          - name: E2E tests on Intel Ponte Vecchio GPU
+            runner: '["Linux", "pvc"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: ext_oneapi_level_zero:gpu;opencl:gpu
+            install_drivers: ${{ contains(needs.detect_changes.outputs.filters, 'drivers') }}
+          - name: E2E tests with dev igc on Intel Ponte Vecchio GPU
+            runner: '["Linux", "pvc"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:${{ contains(github.event.pull_request.labels.*.name, 'ci-no-devigc') && 'latest' || 'devigc' }}
```

Comment:
My comment from the previous PR stands - why this instead of skipping?

Reply:
```
image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:${{ contains(github.event.pull_request.labels.*.name, 'ci-no-devigc') && 'latest' || 'devigc' }}
```
It's the same as for Arc with `dev-igc`. Do we use or intend to use `ci-no-devigc` labels on PR to exclude testing with dev-igc?

Reply:
Well, having the bug before doesn't justify copying it...

Reply:
If our current workflow for Arc + `dev-igc` handles `ci-no-devigc` label incorrectly, then I'll fix that first before commencing with this PR.

Reply:
PR to fix Arc + dev-igc: https://github.com/intel/llvm/pull/15318